### DISCUSSION TITLE
Possibility of providing repo of demo code presented at DrupalSouth?

### DISCUSSION THREAD

**Discussion by McGern** [larowlan/vite-plugin-twig-drupal#27] [General]
Hi,

I found this plugin because of the talk @larowlan gave at DrupalSouth https://www.youtube.com/watch?v=II5OPZaqt0I and think this looks like what I need to support a FE build of components intended to be integrated into Drupal. We are not doing any of the actual Drupal work, but I had intended to hand over files that were a bit closer to what they might be expecting to help smooth the transition.

So was wondering if it would be possible to post a link to the repo that you used during the presentation please? Am a bit stuck on a few of the points demonstrated and seeing the code might help with:-
- How you are compiling files with the css extension as scss (not necessarily a sticking point, because I have scss compiling down to css at the moment
- How you are linking js files to the component so it functions correctly in storybook
- Perhaps how you are dealing with exporting the twig files as plain twig, without compiling - what I have automatically transforms the "component.twig" files into js versions e.g. "component.twig.js" 

Thanks very much,
J

**Comment by larowlan** [comment]
heya, yep its here https://github.com/larowlan/ds-nice-things

**Reply by McGern** [reply]
Thanks very much
